benchmarks/benchmarks/sparse_dataset.py

```diff
@@ -5,8 +5,8 @@
 from scipy import sparse
 
 from anndata import AnnData
+from anndata._core.sparse_dataset import sparse_dataset
 from anndata._io.specs import write_elem
```

Comment:
```suggestion
from anndata import AnnData, sparse_dataset, write_elem
```

Reply:
I _think_ the benchmark needs this file to be stable? https://github.com/scverse/anndata/actions/runs/10680419101/job/29601783819

Reply:
ah of course, if we want to run it with old commits. OK.